src/earthdataVarinfo/handler.py

```diff
@@ -0,0 +1,79 @@
+import os
+import traceback
+import json
+import shutil
+from tempfile import mkdtemp
+
+from varinfo import VarInfoFromNetCDF4
+from varinfo.cmr_search import get_granules, get_granule_link, download_granule
+from varinfo.umm_var import export_all_umm_var_to_json, get_all_umm_var
+
+
+def default(event, context):
+  print('Calling earthdata-varinfo lambda')
+  # print(json.dumps(event, indent=4))
+  # Get authorization token from header
+  headers = event['headers']
+  if 'authorization' in headers:
+    token = headers['authorization'] 
+    # print('Token in python lambda:'+token)
+
+  # Get collectionConceptId from the query parameters
+  query_string_parameters = event['queryStringParameters']
+  if query_string_parameters != None and 'collection_concept_id' in query_string_parameters:
+    collection_concept_id = query_string_parameters['collection_concept_id']
+    # print('collection_concept_id:'+collectionConceptId)
+
+  # Setup CMR search url from cmrRootUrl
+  cmr_url = os.environ['cmrRootUrl']+'/search/'
+
+  temp_dir = None
+
+  try:
+    # Retrieve a list of 10 granules for the collection
+    granules = get_granules(collection_concept_id, cmr_env=cmr_url, token=token)
+
+    # Get the URL for the first granule (NetCDF-4 file):
+    granule_link = get_granule_link(granules)
+    print('Downloading granule from '+granule_link)
+
+    # Make a temporary directory (this may cause permission issues?):
+    temp_dir = mkdtemp()
```

Comment:
How big is this going to get? Lambda does provide some space, and permissions shouldn't be an issue.. here is some information from the docs:

```You can configure each Lambda function with its own ephemeral storage between 512MB and 10,240MB, in 1MB increments. The ephemeral storage is available in each function’s /tmp directory.```

Reply:
Ill check with David and Owen, but since we are downloading HDF files it would probably depend on how big they can get and how fast we would be able to download within our 30 second timelimit.

Reply:
Will set it to 2GB for now.